core/src/scene/sceneLoader.cpp

```diff
@@ -364,7 +375,7 @@ void SceneLoader::loadStyles(YAML::Node styles, Scene& scene) {
         Node urlNode = styleNode["url"];
         if (urlNode) { logMsg("WARNING: loading style from URL not yet implemented\n"); } // TODO
 
-        scene.addStyle(std::unique_ptr<Style>(style));
+        scene.styles().push_back(std::unique_ptr<Style>(style));
```

Comment:
A style with the same name would only be added if: 1. the style used the name of a built-in style, which we check for explicitly above, or 2. if two yaml nodes had the same name, which is invalid yaml and therefore won't ever be parsed.